apistar/formats.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+import re
+
+from apistar.exceptions import ValidationError
+
+DATE_REGEX = re.compile(
```

Comment:
is the idea you subclass apistar.formats.DateFormat and provide your own DATE_REGEX if you want to support multiple or different formats, e.g. `2017/1/20`?

Reply:
I've not gotten as far as making the formats configurable yet. Something along those lines will need to come, yup.

Having said that, in the particular case of date/time/datetime we might only support ISO formats in the core library, and force developers to use a custom `format` class if they want some other datetime behavior.